From c8eab42c0d4fc38bef6c78c29fefeff2fe98ade0 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.jz@renesas.com>
Date: Tue, 20 Apr 2021 18:09:25 +0700
Subject: [PATCH 362/364] arm64: dts: r8a774a1: move ipmmu_vi0 node to
 supported in mmngr driver

IPMMU-VI0 support multimedia, which is implemented in MMNGR driver.
This commit change compatible string of ipmmu_vi0 to be supported in
mmngr driver when ipmmu enabled.

Signed-off-by: Binh Nguyen <binh.nguyen.jz@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a1.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
index 91bf517..54e4fff 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
@@ -1122,7 +1122,7 @@
 		};
 
 		ipmmu_vi0: mmu@febd0000 {
-			compatible = "renesas,ipmmu-r8a774a1";
+			compatible = "renesas,ipmmu-mmu-r8a774a1";
 			reg = <0 0xfebd0000 0 0x1000>;
 			renesas,ipmmu-main = <&ipmmu_mm 9>;
 			power-domains = <&sysc R8A774A1_PD_ALWAYS_ON>;
-- 
2.7.4

